subprojects/core/src/main/java/org/gradle/api/internal/FeaturePreviews.java

```diff
@@ -28,7 +28,7 @@
     public enum Feature {
         IMPROVED_POM_SUPPORT(false),
         GRADLE_METADATA(true),
-        STABLE_PUBLISHING(true);
```

Comment:
Is there any kind of warning that this feature toggle is now useless?

Reply:
Not yet. I discussed it with @ljacomet and we agreed that we should warn when `enableFeature()` is called for a feature that is no longer active (`IMPROVED_POM_SUPPORT`/`STABLE_PUBLISHING`). I'll submit a follow-up PR.